encoding/thrift/doc.go

```diff
@@ -34,11 +34,12 @@
 //
 // In addition to generating code for types specified in your Thrift file,
 // this will generate two packages for each service in the file: a client
-// package and a server package.
+// package, a server package, and a test package.
 //
 // 	myservice
 // 	   |- myserviceclient
 // 	   |- myserviceserver
+// 	   |- myservicetest
```

Comment:
I called it myservicetest because I wanted to leave room for other testing
utilities to go there. I'm imagining we might need a test server, or maybe
something that integrates with the recorder middleware to give readable
recordings; stuff like that. I don't have anything concrete in mind yet.